doc/releases/changelog-dev.md

```diff
@@ -166,6 +166,18 @@
 
 <h3>Improvements 🛠</h3>
 
+* Added `Resources.total_wires` and `Resources.total_gates` properties to the 
+  ``qml.estimator.Resources`` class. Users can more easily access these quantities from the `Resources` object directly.
+  [(#8761)](https://github.com/PennyLaneAI/pennylane/pull/8761)
+
+* Added `PauliHamiltonian.num_terms` property to the ``qml.estimator.PauliHamiltonian`` class.
+  Users can more easily access the total number of terms (Pauli words) from the `PauliHamiltonian` object directly.
+  [(#8761)](https://github.com/PennyLaneAI/pennylane/pull/8761)
+
+* Improved the resource decomposition for the ``~.pennylane.estimator.QROM`` class. The cost has
```

Comment:
Not rendering as intended. I think this needs the ":class:" directive and maybe single backticks
<img width="813" height="56" alt="image" src="https://github.com/user-attachments/assets/de137ebe-8dd6-4a6a-b72c-65ba1ccb7acf" />